lib/tasks/refresh_prisoner_category.rake

```diff
@@ -0,0 +1,15 @@
+namespace :prisoner_category do
+  desc 'Refresh prisoner category'
+  task refresh_data: :environment do
+    profiles = Profile.joins(:person).where.not(people: { latest_nomis_booking_id: nil })
+    count = profiles.count
+    i = 1
+
+    puts "Refreshing #{count} profiles"
+    profiles.find_each do |profile|
```

Comment:
maybe we can utilise `find_index` here instead?
```
profiles.find_each.with_index do |profile, index|
  ...
end
```

Reply:
yep, good shout 👍